airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobCleaner.java

```diff
@@ -1,148 +0,0 @@
-/*
- * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.scheduler.app;
-
-import com.google.common.collect.Sets;
-import io.airbyte.config.WorkspaceRetentionConfig;
-import io.airbyte.scheduler.models.Job;
-import io.airbyte.scheduler.models.JobStatus;
-import io.airbyte.scheduler.persistence.JobPersistence;
-import java.io.File;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.attribute.BasicFileAttributes;
-import java.nio.file.attribute.FileTime;
-import java.time.LocalDateTime;
-import java.time.OffsetDateTime;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.concurrent.atomic.AtomicLong;
-import java.util.stream.Collectors;
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.filefilter.AgeFileFilter;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * The job cleaner is responsible for limiting the retention of files in the workspace root. It does
- * this in two ways. 1. It cleans out all files and directories that are older than the maximum
- * retention date. 2. It cleans out the oldest files before the minimum retention date until it is
- * within the max workspace size.
- */
-public class JobCleaner implements Runnable {
```

Comment:
Even though it is not use, I think that it should be nice to keep this file for https://app.zenhub.com/workspaces/oss-platform-625a01eb010e82001390f49e/issues/airbytehq/airbyte/11869 cc @davinchia WDYT?

Reply:
Sure, I'm okay with keeping this since it could be helpful in the future. I will add a comment linking back to the current state of the code before this PR in case anyone wants to see how this was used

Reply:
I added this back, and moved it into the airbyte-scheduler/persistence module, because I removed the airbyte-scheduler/app directory

Reply:
good catch Benoit!